test/integration/integration_test.go

```diff
@@ -55,6 +55,16 @@ func TestMain(m *testing.M) {
 	var godogPaths = flag.String("paths", "./features", "")
 	flag.Parse()
 
+	if *godogTags != "" {
+		*godogTags += "&&"
+	}
+	runner := util.MinishiftRunner{CommandPath: givenPath}
+	if runner.IsCDK() {
```

Comment:
I guess I am ok, to add these hooks into the code. This will allow us then to add CDK specific tests downstream. 

BTW, should the annotation maybe be names '@cdk-only' and 'minishift-only'?